rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -110,6 +111,7 @@
             }
           },
           "messages": {
+            "patternMessage": "Username must start with a letter, be at least three characters long and only contain alphanumeric characters, dashes and underscores.",
```

Comment:
Add this changes to the commit message

Reply:
`Improves the validation message for the patron username."